packages/manager/apps/pci-savings-plan/src/index.tsx

```diff
@@ -39,13 +39,16 @@ const init = async (appName: string) => {
     // nothing to do
   }
 
-  ReactDOM.createRoot(document.getElementById('root')!).render(
-    <React.StrictMode>
-      <ShellContext.Provider value={context}>
-        <App />
-      </ShellContext.Provider>
-    </React.StrictMode>,
-  );
+  const rootElement = document.getElementById('root');
+  if (rootElement) {
+    ReactDOM.createRoot(rootElement).render(
+      <React.StrictMode>
+        <ShellContext.Provider value={context}>
+          <App />
+        </ShellContext.Provider>
+      </React.StrictMode>,
+    );
+  }
```

Comment:
The previous code throws an error if the root element is not found. Please add an `else` case to know why we have a blank page